phase2/ansible/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM debian:jessie
+
+RUN apt-get update && apt-get install -y git python-dev python-setuptools build-essential curl vim jq libffi-dev libssl-dev
+
+RUN easy_install pip
+RUN pip install paramiko PyYAML Jinja2 httplib2 six pycrypto
+
+RUN git clone git://github.com/ansible/ansible.git --recursive /opt/ansible
```

Comment:
It'd be good to checkout release branch or tag. If we want latest, may be could checkout a specific revision?